scipy/optimize/_nonlin.py

```diff
@@ -1338,8 +1338,8 @@ class KrylovJacobian(Jacobian):
         Note that you can use also inverse Jacobians as (adaptive)
         preconditioners. For example,
 
-        >>> from scipy.optimize.nonlin import BroydenFirst, KrylovJacobian
-        >>> from scipy.optimize.nonlin import InverseJacobian
+        >>> from scipy.optimize import BroydenFirst, KrylovJacobian
+        >>> from scipy.optimize import InverseJacobian
```

Comment:
Do these imports work for you?  When I try this (with SciPy version'1.10.0.dev0+1373.318d8c6'), none of these names can be imported from `scipy.optimize`.

Reply:
https://github.com/scipy/scipy/pull/16532#issuecomment-1173112142 comments crossed